lib/ips/composition.rb

```diff
@@ -25,17 +47,29 @@ class Composition < Inferno::TestGroup
     end
 
     test do
-      title 'Server returns Composition resource that matches the Composition (IPS) profile'
+      title 'Composition resources conform to IPS profile'
       description %(
-        This test will validate that the Composition resource returned from the server matches the Composition (IPS) profile.
+        This test validates that the Composition resource returned from the server
+        conforms to the [Composition (IPS) profile](http://hl7.org/fhir/uv/ips/STU1.1/StructureDefinition/Composition-uv-ips).
+        
+        The IPS Composition profile requirements include:
+        * Must have type code indicating it is an IPS document
+        * Must include mandatory sections (e.g., allergies, medications, problems)
+        * Must use required section codes from the IPS document code system
+        * Must reference a Patient as the subject
+        * Must include author information
+        * Must have a status and date
+        
+        For detailed requirements, refer to the [IPS Implementation Guide - Composition](http://hl7.org/fhir/uv/ips/STU1.1/composition.html).
+        The Composition serves as the foundation of the IPS document, organizing all relevant
+        clinical information into a structured format.
       )
-      # link 'http://hl7.org/fhir/uv/ips/StructureDefinition/Composition-uv-ips'
+      
       uses_request :composition
 
       run do
-        assert_valid_resource(profile_url: 'http://hl7.org/fhir/uv/ips/StructureDefinition/Composition-uv-ips')
+        assert_valid_resource(profile_url: 'http://hl7.org/fhir/uv/ips/STU1.1/StructureDefinition/Composition-uv-ips')
```

Comment:
This is wrong. canonical urls don't put versions in the path like this.

If we want to be specific about the version in here, technically we should be doing `http://hl7.org/fhir/uv/ips/StructureDefinition/Composition-uv-ips|1.1.0` everywhere.  We kinda get that for 'free' as long as v1.1.0 is the only one being loaded in the validator, but i suppose we should be explicit everywhere about that if our test descriptions say that we are specifically checking stu1.1.

i think for this, just removing the STU1.1 in the middle and not worrying about managing a version in here is the way to go.